msu/systems/serialization/load.nut

```diff
@@ -27,5 +30,6 @@ includeFile("serialization_mod_addon.nut");
 		_get = @(_k)@(...)""
 	});
 	worldState.onBeforeSerialize(serEm);
+	::MSU.System.Serialization.SerializationMetaData = serEm.getMetaData();
```

Comment:
I'm not having issues without this line so I believe it is redundant due to the hook on onBeforeSerialize in world_state